prow/cmd/deck/static/pr-script.js

```diff
@@ -969,19 +1032,77 @@ function createPRCard(pr, builds = [], queries = [], tidePools = []) {
         });
         score = (labels.length + missingLabels.length > 0) ? score
             / (labels.length + missingLabels.length) : 1.0;
-        processedQuery.push(
-            {score: score, labels: labels, missingLabels: missingLabels});
+        processedQueries.push(
+            {score: score, labels: labels, missingLabels: missingLabels, excludedBranches:query.excludedBranches,
+                includedBranches: query.includedBranches, milestone:query.milestone});
     });
     // Sort queries by descending score order.
-    processedQuery.sort((q1, q2) => {
+    processedQueries.sort((q1, q2) => {
+        let q1Excluded = 0, q2Excluded = 0;
+        if (pr.BaseRef && pr.BaseRef.Name &&
```

Comment:
This function is more verbose than it needs to be which makes it hard to read. Lets check `pr.BaseRef && pr.BaseRef.Name` just once.